### PR TITLE
Budget app: Create budget app

### PR DESCRIPTION
**Pull request by julie-ify**
## Here are all the interactions I implemented
- Splash screen

  - A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.

- Sign up and log in pages
  - The user should be able to register in the app with full name, email and password (all mandatory).
  - The user can log into the app using email and password.
  - If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).
  - Home page (catagories page)

- When the user logs in, they are presented with the categories page.
  - For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
  - When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
  - There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
  - Transactions page

- For a given category, the list of transactions is presented, ordered by the most recent.
  - At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in 
  - that category).
  - There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

- "Add a new category" page
  - The user fills out a form to create a new category, indicating their name and icon (both mandatory).
  - The user click (or taps) the "Save" button to create the new category, and is taken to the home page on success.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

- "Add a new transaction" page
  - The user fills out a form to create a new transaction with:
  - name (mandatory)
  - amount (mandatory)
  - categories (mandatory at least one)
  - The user click (or taps) the "Save" button to create the new category, and is taken to the transactions page for that category.
  - When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.

Here is [link ](https://www.loom.com/share/25d0142d8c6a4eabbafccec9af3158cb)to my presentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,91 @@
...
+# Demo Link
 
-* System dependencies
+Click [here](https://julie-budgetapp.herokuapp.com/) to view the website online.
 
-* Configuration
+# Video presentation Link
 
-* Database creation
+Click [here](#)
```
Comment (rloterh) [inline README.md]: The provided video link to this Pull Request does is invalid. Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project. The recorded video should:
- [ ]  entail the **features** of the project.
- [ ] have a maximum length of **5 minutes**.
- [ ] be **clear and objective description** about the project in professional English. The topic and purpose of the project are made clear.
- [ ] contains a demo of the project, highlighting all the project features.
- [ ] contain a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Juliana, 

You did great on this project, and for making the changes requested in the previous review(s)! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍
- Descriptive Pull Request
- Project correctly deployed and working
- App is working as intended
- Test cases are passing
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (julie-ify) [reply README.md]: I have included the video presentation link to README. Thank

**Review by peterrobert — APPROVED** [review]
Hi @julie-ify, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**